### ISSUE TITLE
Pin Version in setup.py

### ISSUE THREAD

**Issue by simon-mo** [ucbrise/clipper#587]
Currently we are not pining version for some of our `clipper_admin` dependencies. We should pin them down. Never use '*'

**Comment by withsmilo** [comment]
Yes, we need to update the version of libraries used by Clipper also.
